packages/react-button/src/components/ToggleButton/checkedBehaviorDefinition.ts

```diff
@@ -0,0 +1,30 @@
+import { BehaviorRule, Rule } from '@fluentui/a11y-testing';
+
+export const checkedBehaviorDefinition: Rule[] = [
+  BehaviorRule.root()
+    .doesNotHaveAttribute('aria-pressed')
+    .description('by default'),
+  BehaviorRule.root()
+    .afterEvent(
```

Comment:
could you also add afterClick(), so that we do not have to create the mouse event in every test?

Reply:
yeah that makes sense, it can be extended later for other handlers we may add 👍